### PR TITLE
fix: assemble executable jar for pd and store

### PR DESCRIPTION
**Pull request by VGalaxies**
## Purpose of the PR

fix https://github.com/apache/incubator-hugegraph/pull/2275

refer to https://hugegraph.feishu.cn/wiki/UfL0w2m8Aiekl9kHsedc8jQKnN7#QuiKdmz2ZoY3WMxqkiaczvc6nKb

## Main Changes

Due to the change of `hugegraph-pd`'s parent from `spring-boot-starter-parent` to `hugegraph`, and `hugegraph`'s parent being `apache`, along with modifications to the `artifactId` of certain sub-modules (for example, `hugegraph-pd` to `hg-pd-service`), the following steps need to be taken:

1. Modify the descriptor of the `maven-assembly-plugin`, which involves adjusting the collected jar package names accordingly.
2. Update the corresponding jar package names in the startup script.
3. Since `hugegraph-pd` is started using Spring Boot, the `spring-boot-maven-plugin` needs to be reconfigured.

## Verifying these changes

- [x] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [x]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

**Comment by imbajin** [comment]
could also refer the toolchain PR:
https://github.com/apache/incubator-hugegraph-toolchain/commit/025c5ab86be0bb4f3a0c4c1d1d8834580715388a#diff-a97944bbb4e440a8f7f8ef46f83479ecaa1cd8da2fb1e88437b2acab9d8f842f

File: hugegraph-pd/hg-pd-service/pom.xml [hunk 1]
```diff
@@ -138,11 +144,13 @@
...
                     <execution>
                         <configuration>
-                            <classifier>spring-boot</classifier>
                             <mainClass>
                                 org.apache.hugegraph.pd.boot.HugePDServer
                             </mainClass>
                         </configuration>
+                        <goals>
+                            <goal>repackage</goal>
+                        </goals>
```
Comment (imbajin) [inline hugegraph-pd/hg-pd-service/pom.xml:153]: we need some comment here?

File: hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh [hunk 2]
```diff
@@ -115,7 +115,7 @@ echo "Starting HugeGraphPDServer..."
 
 # Turn on security check
 exec ${JAVA} ${JAVA_OPTIONS} -jar -Dspring.config.location=${CONF}/application.yml \
```
Comment (imbajin) [inline hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh:117]: could also add `process name` param for it..... refer server/toolchain/computer, so as `store`

Comment (VGalaxies) [reply hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh]: add `process name` param for `pd`

`store` already added

File: hugegraph-pd/hg-pd-service/pom.xml (hunk 1 above)
Comment (imbajin) [reply hugegraph-pd/hg-pd-service/pom.xml]: > refer to [docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging](https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging)

fine, better to add url refer for it in future, currently leave this in PR context

**Pull request merged**